api/data.go

```diff
@@ -105,14 +106,19 @@ func (api *API) GetCacheTime(ctx context.Context, w http.ResponseWriter, req *ht
 
 	cacheTime, err := api.dataStore.GetCacheTime(ctx, id)
 	if err != nil {
-		log.Error(ctx, "error retrieving cache time from datastore", err)
-		http.Error(w, "Internal Server Error", http.StatusInternalServerError)
+		if err == errs.ErrCachetimeNotFound {
+			log.Error(ctx, "getCacheTime endpoint: api.dataStore.GetCacheTime document not found", err)
```

Comment:
We shouldn't be logging with a level of "Error" here, because an item not found is part of the normal workflow. This should be lowered to an "Info", I would say.